scss/components/_emergency-banner.scss

```
@@ -1,176 +1,135 @@
+  padding: 27px 0;
+}
+
 .emergency-banner {
-    font-family: "Open Sans",Helvetica,Arial,sans-serif;
-    -webkit-font-smoothing: antialiased;
-    -moz-osx-font-smoothing: grayscale;
-    font-weight: 400;
-    font-size: 16px;
-    line-height: 1.25;
-    background-color: #b1b4b6;
-    color: #fff;
-    margin-top: 2px;
-    padding: 15px 0
-}
-@media print {
-    .emergency-banner {
-        font-family: sans-serif
-    }
-}
-@media (min-width: 40.0625em) {
-    .emergency-banner {
-        font-size:19px;
-        line-height: 1.3157894737;
-    }
-}
-@media print {
-    .emergency-banner {
-        font-size: 14pt;
-        line-height: 1.15
-    }
-}
-@media (min-width: 40.0625em) {
-    .emergency-banner {
-        padding:30px 0
-    }
-}
-.emergency-banner {
-    border-bottom: 5px solid #fff;
-    border-top: 5px solid #fff;
-    margin-bottom: -10px;
-    margin-top: 0;
-    position: relative;
-    z-index: 10
-}
-.emergency-banner__heading {
-    font-family: "nta",arial,sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 1.25;
+  background-color: #b1b4b6;
+  color: $white;
+  margin-top: 2px;
+  padding: 18px 0;
+  border-bottom: 5px solid $white;
+  margin-bottom: -10px;
+  margin-top: 0;
+  position: relative;
+
+  &__heading {
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
     font-weight: 700;
-    font-size: 32px;
-    line-height: 1.1111111111;
+    font-size: 30px;
+    line-height: 42px;
     margin: 0;
     padding: 0;
-}
-@media print {
-    .emergency-banner__heading {
-        font-family: sans-serif
-    }
-}
-@media (min-width: 40.0625em) {
-    .emergency-banner__heading {
-        font-size:32px;
-        line-height: 1.25
-    }
-}
-@media print {
-    .emergency-banner__heading {
-        font-size: 18pt;
-        line-height: 1.15;
-    }
-}
-.homepage .emergency-banner__heading {
-    font-family: "nta",arial,sans-serif;
-    -webkit-font-smoothing: antialiased;
-    -moz-osx-font-smoothing: grayscale;
-    font-weight: 700;
-    line-height: 1.09375
-}
-@media print {
-    .homepage .emergency-banner__heading {
-        font-family: sans-serif
-    }
-}
-@media (min-width: 40.0625em) {
-    .homepage .emergency-banner__heading {
-        font-size:48px;
-        line-height: 1.0416666667;
-    }
-}
-@media print {
-    .homepage .emergency-banner__heading {
-        font-size: 32pt;
-        line-height: 1.15;
-    }
-}
-@media (min-width: 40.0625em) {
-    .homepage .emergency-banner__heading {
-        margin-bottom:20px;
-    }
-}
-.emergency-banner__description {
-    font-family: "nta",arial,sans-serif;
+    @include breakpoint(md-max) {
+      font-size: 26px;
+      line-height: 36.4px;
+    }
+  }
+  .page-type--homepage &__heading {
+    font-size: 36px;
+    line-height: 50.4px;
+    @include breakpoint(md-max) {
+      font-size: 28px;
+      line-height: 39.2px;
```

can you please change this to 140 as those points will be most likely rounded
```suggestion
      line-height: 140;
```

---

You mean 1.4? I'll change the values on this and the others